lib/document-to-html.js

```diff
@@ -145,6 +145,9 @@ function DocumentConversion(options, comments) {
         if (run.isBold) {
             paths.push(findHtmlPathForRunProperty("bold", "strong"));
         }
+        if (run.isSmallCaps) {
+            paths.push(findHtmlPathForRunProperty("smallCaps"));
+        }
```

Comment:
To keep the order consistent, could this go before strikethrough instead? (That way the nesting order matches the order in the object.)